siswrap/handlers.py

```diff
@@ -1,41 +1,56 @@
-import tornado.ioloop
-import tornado.web
 import jsonpickle
-from .siswrap import ProcessService, Logger, ProcessInfo, Wrapper
-from .configuration import ConfigurationService
-import os
-import click
+import arteria
+from arteria.web.handlers import BaseRestHandler
+from arteria.web.state import State
+from wrapper_services import ProcessService, Wrapper, ProcessInfo
 
 
-class BaseHandler(tornado.web.RequestHandler):
-    """ Our base Tornado process handler.
+class BaseSiswrapHandler(BaseRestHandler):
+    """ Provides core logic for Siswrap handlers
     """
 
     HTTP_OK = 200
     HTTP_ACCEPTED = 202
     HTTP_ERROR = 500
 
+    # FIXME: This should probably be documented in arteria core.
```

Comment:
Yes, exactly. E.g. my handlers need to be able to access the config and process services.